packages/xo-server/src/api/xostor.mjs

```diff
@@ -85,12 +85,14 @@ updateDependencies.resolve = {
   host: ['host', 'host', 'administrate'],
 }
 
-export async function formatDisks({ disks, force, host, ignoreFileSystems, provisioning }) {
+export async function formatDisks({ disks, force, host, ignoreFileSystems, provisioning, xapi }) {
   const rawDisks = disks.join(',')
   return Task.run(
     { properties: { disks, name: `format disks on ${host.name_label}`, objectId: host.uuid } },
     async () => {
-      const xapi = this.getXapi(host)
+      if (xapi === undefined) {
+        xapi = this.getXapi(host)
+      }
```

Comment:
As it said in the PR description:
```
The xapi parameter is added to formatDisks function, to avoid triggering the error no connection found for object <host-uuid> if the master toolstack has been restarted
```